pages/profiles/graph.svg.js

```diff
@@ -0,0 +1,29 @@
+// node_modules
+import Image from "next/image";
+
+// components
+import Breadcrumbs from "../../components/breadcrumbs";
+import PagePreamble from "../../components/page-preamble";
+import { DataPanel, DataAreaTitle } from "../../components/data-area";
+
+export default function GraphSvg() {
+  return (
+    <>
+      <Breadcrumbs />
+      <PagePreamble />
+      <>
+        <DataAreaTitle>Graph</DataAreaTitle>
+        <DataPanel>
+          <a href="https://api.data.igvf.org/profiles/graph.svg">
+            <Image
+              src="https://api.data.igvf.org/profiles/graph.svg"
+              height={5356}
+              width={4703}
+              alt="IGVF graph"
+            />
```

Comment:
I see where you got the width and height, but as they add and modify schemas, the SVG’s inherent height will change and the image might get cut off or squeezed. Sometimes `<Image>` causes more difficulty than it solves for flexible things like this. I’d recommend this instead:
```
const graphUrl = `${API_URL}/profiles/graph.svg`;
…
<a href={graphUrl} className="block">
  <picture className="block">
    <img src={graphUrl} alt="IGVF graph" />
  </picture>
</a>
```
Using `<img>` gets you an ESLint error unless you wrap it in a `<picture>` element.